scripts/run_etcd.sh

```diff
@@ -1,12 +1,28 @@
 #!/bin/bash
 
-mkdir -p external
```

Comment:
@AdoHe What do you think about separating this into another PR? We don't need to block this.